### ISSUE TITLE
Monitrored Status color incorrect on author if file count > book count

### ISSUE THREAD

**Issue by plmcgrn** [Readarr/Readarr#1836]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When an author has books that are chapterized MP3's, the color for the author's Monitored Status will be red, instead of blue, even if all monitored books are on disk.

### Expected Behavior

Monitored status color should be blue, indicating all monitored books are downloaded.

### Steps To Reproduce

1. Have author with at least one monitored book
2. Have that book be a multi-file MP3 audiobook
3. Monitored Status color will be red

### Environment

```markdown
- OS: MacOS 12.5.1
- Readarr: 0.1.1.1320
- Docker Install: yes
- Using Reverse Proxy: no
- Browser: Chrome 104.0.5112.101
```

### What branch are you running?

Develop

### Trace Logs?

n/a, this is purely a cosmetic UI issue

[AB#3666](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3666)

**Comment by tomatosoupcan** [comment]
Also experiencing this, I think the main issue is that the system count files at all. Files present out of books monitored is comparing two different units. Ideally I'd think it should be non-missing books out of books monitored, or at least the option for it.

**Comment by fernandossmm** [comment]
Agree, this is not how it should work. However, it seems like a quicker fix to first change the color rather than change the check. It works fine enough in the Author view, but not in the Books view.
![image](https://github.com/Readarr/Readarr/assets/36533185/3d200e51-43d8-4b1a-aa75-ece67d019848) ![image](https://github.com/Readarr/Readarr/assets/36533185/5f2854a3-5707-4102-84dc-533a522eb826)

**Comment by mynameisbogdan** [comment]
I think I fixed this with d659e86a7d908479104311819a7232cf0c3c6d6d already.
